qa/integration-tests/src/test/java/io/camunda/it/backup/BackupDBClient.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.backup;
+
+import io.camunda.search.connect.configuration.DatabaseType;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+
+/** interface to abstract Elasticsearch/Opensearch clients for testing purposes */
+public interface BackupDBClient extends AutoCloseable {
```

Comment:
🔧I know it's test only, but I would document the contract in case we need to implement it again in the future.